runtime/plugins/comment/comment.lua

```diff
@@ -89,7 +89,7 @@ end
 
 function commentLine(bp, lineN, indentLen)
     local line = bp.Buf:Line(lineN)
-    local commentType = bp.Buf.Settings["commenttype"]
+    local commentType = bp.Buf.Settings["comment.type"]
```

Comment:
Instead of merging the `master` branch into the PR branch it is more preferred to rebase the PR branch against the actual upstream `master`.
This allows as to extract a clean set of patches, which can be applied easily locally.

This also helps to cleanup the PR's history.